client.go

```diff
@@ -132,6 +132,10 @@ type Client interface {
 	LSPSetDynamicAddresses(lsp string, address string) (*OvnCommand, error)
 	// Get dynamic addresses from LSP
 	LSPGetDynamicAddresses(lsp string) (string, error)
+	// Set external_ids for LSP
+	LSPSetExternalIds(lsp string, external_ids map[string]string) (*OvnCommand, error)
+	// Get external_ids from LSP
+	LSPGetExternalIds(lsp string) (map[string]string, error)
```

Comment:
Same here, no need to add the Get interface for external_ids

Reply:
`GetExternalIds` is essential because external_ids are saved as `string[interface{}]interface{}`. The implementation returns a map[string]string. It's implementing some type conversion logic.

Reply:
ok